de.fu_berlin.inf.dpp.core/src/de/fu_berlin/inf/dpp/communication/extensions/UserListExtension.java

```diff
@@ -34,19 +34,22 @@ public void addUser(User user, long flags) {
 
         public long flags;
         public JID jid;
+        public boolean isServer;
         public int colorID;
         public int favoriteColorID;
```

Comment:
![Vulnerability](https://sonarsource.github.io/sonar-github/vulnerability.png 'Vulnerability') Vulnerability: Make favoriteColorID a static final constant or non-public and provide accessors if needed. ([squid:ClassVariableVisibilityCheck](https://sonarcloud.io/organizations/saros/rules#rule_key=squid%3AClassVariableVisibilityCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=saros_complete&pullRequest=284&issues=AWc2MMo10EcSutaxS9dn&open=AWc2MMo10EcSutaxS9dn)

[ISSUE_KEY]: AWc2MMo10EcSutaxS9dn